backend/Origam.Rule/DatasetRuleHandler.cs

```diff
@@ -265,11 +265,7 @@ public void OnRowCopied(DataRow row, IDataDocument data, DataStructureRuleSet ru
 
 		private void table_RowChanged(object sender, DataRowChangeEventArgs e)
 		{
-			// The RowChanged event fires after every call to DataRow.EndEdit().
-			// Every field change calls the DataRow.EndEdit(). This led to 
-			// execution of the OnRowChanged handler after changing just a single
-			// field which added undesired overhead.
-			if (e.Action != DataRowAction.Change && e.Action != DataRowAction.Nothing && e.Action != DataRowAction.Commit)
+			if (e.Action != DataRowAction.Nothing && e.Action != DataRowAction.Commit)
```

Comment:
Will this break the original test case? UpdateObject on master-detail with 20 000 details...

Reply:
Yes, but this was not the right solution. I will review it again.